module/project/ProjectDialog.kt

```diff
@@ -155,43 +155,17 @@ object ProjectDialog {
                 Form.IconButton(
                     icon = Icon.FOLDER_OPEN,
                     tooltip = Tooltip.Arg(Label.OPEN_PROJECT_DIRECTORY)
-                ) { launchDirectorySelector(state, window, title) }
+                ) {
+                    val selectedDirectoryPath = selectFilePath(window, Label.OPEN_PROJECT_DIRECTORY, SelectorOptions.DIRECTORIES_ONLY)
```

Comment:
That makes sense. 

I'm still enthusiastic about having the function return the path selected (or null if one isn't) rather than having some stateful file dialog stick around and having to do the work everywhere we make use of it to determine whether nothing was selected by probing the `directory` and `file` values rather than just a single returned value.

On this basis I've gone with `SelectFileDialog.selectPath()`.